travel_platform/izi_travel_client.py

```diff
@@ -1,6 +1,8 @@
+import os
```

Comment:
It would be great to add some folder like `api` and add all our APIs functionalities into this.